app/lib/fake/server/fake_storage_server.dart

```diff
@@ -78,7 +78,7 @@ class FakeStorageServer {
       await for (final m in stream) {
         final disposition = _parse(m.headers['content-disposition']);
         final name = disposition['name']!;
-        final data = await readAsBytes(m);
+        final data = await readByteStream(m);
```

Comment:
For this purpose there is also:
https://pub.dev/documentation/async/latest/async/collectBytes.html

I didn't use that in the `TaskBackend` because I wanted something I could give a maximum number of bytes.